File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/run_list_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RunListResponse:
    """OpenAI run list response"""
    object: str = None
    data: List[RunResponse] = None
    has_more: bool = None
    first_id: str = None
    last_id: str = None

File: sdkwork-ai-sdk-typescript/src/types/run-list-response.ts
import type { RunResponse } from './run-response';

/** OpenAI run list response */
export interface RunListResponse {
  /** Object type */
  object?: string;
  /** Run list */
  data?: RunResponse[];
  /** Whether more data exists */
  has_more?: boolean;
  /** First item id */
  first_id?: string;
  /** Last item id */
  last_id?: string;
}
